File: support_tools/swigwin-3.0.12/Examples/test-suite/csharp/sneaky1_runme.cs
using System;
using sneaky1Namespace;

public class runme
{
    static void Main() 
    {
        if (sneaky1.add(30, 2) != 32)
            throw new Exception("add test failed");

        if (sneaky1.subtract(20, 2) != 18)
            throw new Exception("subtract test failed");

        if (sneaky1.mul(20, 2) != 40)
            throw new Exception("mul test failed");

        if (sneaky1.divide(20, 2) != 10)
            throw new Exception("div test failed");
    }
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/ocaml/sneaky1_runme.ml
(* Stolen from ruby test cases *)

open Swig
open Sneaky1

let x = Sneaky1._add (C_list [ C_int 3; C_int 4 ])
let y = Sneaky1._subtract (C_list [ C_int 3; C_int 4 ])
let z = Sneaky1._mul (C_list [ C_int 3; C_int 4 ])
let w = Sneaky1._divide (C_list [ C_int 3; C_int 4 ])
